cadquery/occ_impl/shapes.py

```diff
@@ -809,12 +810,20 @@ def assembleEdges(cls, listOfEdges):
         """
             Attempts to build a wire that consists of the edges in the provided list
             :param cls:
-            :param listOfEdges: a list of Edge objects
+            :param listOfEdges: a list of Edge objects. The edges are not to be consecutive.
```

Comment:
Does "The edges are not to be consecutive" here mean that the call will fail if they are consecutive, or is it just saying that they don't have to be consecutive?

Reply:
@jmwright 3 out of the 5 examples I made for ```BRepOffsetAPI_MakeFilling``` #253 have non consecutive edges, so I guess it does not matter.

Reply:
Do you want me to add one non consecutive edges case to the tests?

Reply:
Yes, please. Based on all the bugs we've fought over the years with edge ordering I think it's worth it.

Reply:
@jmwright I have added 3 more tests in TestsAssembleEdges.py with consecutive and non consecutive edges. I modified runtests.py to check that it is running, but now I have one conflict reported. Maybe I should not have included this test in runtests.py?